Update nginx.rst, mising reload, some more explanation (#1280)
minvws/nl-kat-coordination: docs/source/technical_design/nginx.rst

    $ nano /etc/nginx/sites-available/openkat.example.com

> Then put the following content in there, but insert the domain name openkat.example.com into the domainname you are pointing to this Openkat install:

Reviewer: ```suggestion
Then put the following content in there, but insert the domain name openkat.example.com into the domain name you are pointing to this OpenKAT install:
```

> Then put the following content in there, but insert the domain name openkat.example.com into the domain name you are pointing to this OpenKAT install:

Reviewer 2: ```suggestion
Then put the following content in there, but replace the domain name `openkat.example.com` with the domain name you are using for this OpenKAT install:
```

    }

With this, we have set up minimal things for OpenKAT. Much more is possible, but this is the minimum we need to use OpenKAT properly.

> we assume that OpenKAT listens on the server to port 8000, you could check this by doing 'wget -O- localhost:8000' it should show you some html output.

Reviewer: ```suggestion
we assume that OpenKAT listens on the server to port 8000, you could check this by doing `wget -O- localhost:8000` it should show you some html output.
```

Logging
=======

> We write the log files in the directory /var/log/nginx/. If you rename the log files (in particular, give them an extension other than .log) or put them in a different place, you will have to reconfigure Logrotate for this as well, otherwise the log files will continue to grow indefinitely. For this, see the configuration in the file /etc/logrotate.d/nginx

Reviewer: ```suggestion
We write the log files in the directory `/var/log/nginx/`. If you rename the log files (in particular, give them an extension other than .log) or put them in a different place, you will have to reconfigure Logrotate for this as well, otherwise, the log files will continue to grow indefinitely. For this, see the configuration in the file `/etc/logrotate.d/nginx`
```

If everything is okay, it will report it that way. If there is an error in the configuration (because you forgot an ; somewhere, for example), it will show you the line number where the problem is near. Note: So you might need to add an ; on the previous line.

.. code-block:: sh

    $ service nginx reload

SSL certificates
================

With this basic configuration, we can then let Certbot arrange an SSL certificate; Certbot itself will also take care of setting this up in your web server configuration.

> Before we can setup a certificate, you need to make sure the domainname you used in the earlier config points to the external IP address for the host running nginx.

Reviewer: ```suggestion
Before we can setup a certificate, you need to make sure the domain name you used in the earlier config points to the external IP address for the host running nginx.
```
